test/test_helper.rb

```diff
@@ -47,9 +47,8 @@ def serialization_options(options)
 
 require 'support/rails5_shims'
 
-require 'fixtures/active_record'
-
-require 'fixtures/poro'
+require 'support/fixtures/active_record'
+require 'support/fixtures/poro'
```

Comment:
I like to try to keep the number of folders per folder reasonably lowish. I guess the argument for moving to fixtures also could be "the tests don't *require* fixtures, it just makes reusing objects a little more DRY"